include/uapi/sound/sof/ext_manifest.h

```diff
@@ -63,7 +66,8 @@ enum sof_ext_man_elem_type {
 struct sof_ext_man_elem_header {
 	uint32_t type;		/*< SOF_EXT_MAN_ELEM_ */
 	uint32_t size;		/*< in bytes, including header size */
-	uint8_t blob[];		/*< type dependent content */
+
+	/* just after this header should be type dependent content */
 } __packed;
```

Comment:
To avoid further fixups, I want confirmation that the FW side is merged.

Reply:
@ktrzcinx Why are you removing this field? Without it getting to the data would become more difficult. Now you can just do `hdr->blob`, after this patch this won't be any longer possible. Above you did keep a variable-length `.elements[]` array, why are you removing `.blob[]` here?

Reply:
@lyakh https://github.com/thesofproject/sof/pull/2427#discussion_r394436447. Moreover `hdr->blob` shouldn't be used at all, `hdr` should be every time casted to `type` specific struct. Field `.elements[]` should be deleted in a same way as `.blob[]`, my mistake.